app/presenters/base.presenter.js

```diff
@@ -0,0 +1,19 @@
+class BasePresenter {
+  constructor (data) {
+    this._data = data
+  }
+
+  call () {
+    return this._presentations(this._data)
+  }
+
+  _presentations () {
```

Comment:
Just wondering about the name. As we're returning a single object what do you think of making it singular, for example, `_presentation()`.